disnake/message.py

```diff
@@ -1576,10 +1576,10 @@ async def edit(
                 Supports passing ``None`` to clear attachments.
 
         suppress_embeds: :class:`bool`
-            Whether to suppress embeds for the message. This removes
-            all the embeds if set to ``True``. If set to ``False``
-            this brings the embeds back if they were suppressed.
-            Using this parameter requires :attr:`~.Permissions.manage_messages`.
```

Comment:
The permission requirement should be kept here

Reply:
https://github.com/DisnakeDev/disnake/pull/834/commits/fa6377dce151a49330d6c9042ea96f448a40176e